salt/utils/msgpack.py

```diff
@@ -82,8 +80,10 @@ def _sanitize_msgpack_unpack_kwargs(kwargs):
     https://github.com/msgpack/msgpack-python/blob/master/ChangeLog.rst
     """
     assert isinstance(kwargs, dict)
+    if version >= (0, 5, 2):
```

Comment:
Would be best to define this as versions >= 0.5.2 and <1.0.0 to ensure we do not hit this if statement when running versions >= 1.0.0